employees/services/crud.py

```diff
@@ -0,0 +1,109 @@
+import json
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+class OrganizationManager:
+
+    def create_organization(self, name: str, display_name: str, primary_color: str =  '#eb4034', background_color: str = '#e0b9b6'):
+
+        url = f"https://{os.getenv('TENANT_DOMAIN')}.eu.auth0.com/api/v2/organizations"
+
+        payload = json.dumps({
+            "name": f"{name}",
+            "display_name": f"{display_name}",
+            "branding": {
+                "colors": {
+                    "primary": f"{primary_color}",
+                    "page_background": f"{background_color}"
+                }
+            },
+            "metadata": {},
+            "enabled_connections": [
+                {
+                    "connection_id": f"{os.getenv('DATABASE_ID_CONNECTION')}",
+                    "assign_membership_on_login": True,
+                    "show_as_button": True,
+                    "is_signup_enabled": True
+                }
+            ]
+        })
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+            'Authorization': f"Bearer {os.getenv('MANAGEMENT_API_TOKEN')}"
+        }
+
+        response = requests.request("POST", url, headers=headers, data=payload)
```

Comment:
co z try-except dla requests?

Reply:
done by implementing new function 'make_request_with_error_handling' which is used to do request to auth0 and avoid repeating the same try/except code